### DISCUSSION TITLE
Custom Badge Colors

### DISCUSSION THREAD

**Discussion by kassbohm** [melissawm/sphinx-tags#109] [Ideas]
Badge colors can be set matching the "PyData Theme". Wouldn't it be nice to let the user (of sphinx_tags) define badge colors? I did not find anything about that the user has this option in the sphinx_tags documentation.

**Comment by JWCook** [comment]
Yes, there is a feature for this! https://sphinx-tags.readthedocs.io/en/latest/configuration.html#badge-colors

Since the badges are using sphinx-design, there is already a list of semantic colors that will use the colors defined for your Sphinx theme (PyData, in your case). They are [listed here](https://sphinx-design.readthedocs.io/en/latest/badges_buttons.html#badges).

**Comment by kassbohm** [comment]
There is a misunderstanding.

I know the feature you indicated with the link.

But I am **not** using the **PyData** theme. **Nor** am I using the **sphinx-design**.

I think, it would be nice if the user could set any color.

**Comment by JWCook** [comment]
You can set those colors to whatever you want in your theme's Sphinx config, but the syntax can vary by theme. With [Furo](https://pradyunsg.me/furo/customisation/colors/), for example, it would look like:
```py
# Theme color config
html_theme_options = {
    "light_css_variables": {
        "color-brand-primary": "#7C4DFF",
    },
}

# Badge color config (references theme colors)
tags_badge_colors = {
    "tag1": "primary",
}
```

**Reply by kassbohm** [reply]
This works only for those themes, which have these options... e.g. PyData and Furo. 

But you cannot set those colors through this e.g. for the Readthedocs theme.

**Comment by story645** [comment]
Is there a way to set the color w/o having to overwrite one of the categories?

**Reply by JWCook** [reply]
Not that I know of; I believe that would require changes in sphinx-design.

**Reply by melissawm** [reply]
That is also my understanding. This could probably be solved with some css though!
